<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QOTAEMV" ignored="0" name="AG" tests="2" failures="1" timestamp="19 Feb 2020 06:07:57 GMT" time="0.054" errors="0">
  <testcase name="TC_003" time="0.008" classname="test.PFDP"/>
  <testcase name="TC_004" time="0.0" classname="test.PFDP">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at test.PFDP.TC_004(PFDP.java:18)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- TC_004 -->
</testsuite> <!-- AG -->
